tests/test_go.py

```diff
@@ -81,8 +81,8 @@ def test_build_helm(auto_container_per_test, container_git_clone):
     """
 
     go_version = auto_container_per_test.connection.check_output("go version")
-    if "go1.20" in go_version or "go1.21" in go_version:
-        pytest.skip("Helm requires Go >= 1.22")
+    if "go1.21" in go_version or "go1.22" in go_version:
+        pytest.skip("Helm requires Go >= 1.23")
```

Comment:
This is incorrect, we're not checking out the main branch of helm, we're checking out the 3.15.1 version